front-end/src/components/User.js

```diff
@@ -30,10 +30,11 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const User = ({ username }, { userId }) => {
```

Comment:
This was the original offending line..... (in `App.js`it's passed as `{ userId }, { username }`). That's really the only change functionally here.

Reply:
Yea you could also do `{username, userId}` since props is one object and so that's why { username } is coming back undefined.

Reply:
ahhh right, makes sense 👍🏾